simde/x86/avx2.h

```diff
@@ -1844,6 +1844,32 @@ simde_mm256_unpacklo_epi16 (simde__m256i a, simde__m256i b) {
 #  define _mm256_unpacklo_epi16(a, b) simde_mm256_unpacklo_epi16(a, b)
 #endif
 
+SIMDE__FUNCTION_ATTRIBUTES
+simde__m256i
+simde_mm256_unpacklo_epi32 (simde__m256i a, simde__m256i b) {
+#if defined(SIMDE_AVX2_NATIVE)
+  return _mm256_unpacklo_epi32(a, b);
+#else
+  simde__m256i_private
+    r_,
+    a_ = simde__m256i_to_private(a),
+    b_ = simde__m256i_to_private(b);
+
+#if defined(SIMDE__SHUFFLE_VECTOR)
+  r_.i32 =SIMDE__SHUFFLE_VECTOR(32, 32, a_.i32, b_.i32,
+                                0, 8, 1, 9, 4, 12, 5, 13);
+#else
+  r_.m128i[0] = simde_mm_unpacklo_epi32(a_.m128i[0], b_.m128i[0]);
+      r_.m128i[1] = simde_mm_unpacklo_epi32(a_.m128i[1], b_.m128i[1]);
```

Comment:
These two lines should line up.

Reply:
I will make sure to check proper indentation from next time. Usually my IDE collapse parts of code, so I may have missed it.